spec/components/hint_spec.rb

```diff
@@ -8,6 +8,18 @@ def component_name
   it "renders hint" do
     render_component(text: "For example, ‘QQ 12 34 56 C’.")
 
-    assert_select(".govuk-hint", text: "For example, ‘QQ 12 34 56 C’.")
+    assert_select '.govuk-hint.govuk-\!-margin-bottom-3', text: "For example, ‘QQ 12 34 56 C’."
+  end
+
+  it "applies a specified bottom margin" do
```

Comment:
Should we add a test which makes sure the class is not applied if you pass a number that's not 0-9?

Reply:
Yes, good point. Had to rewrite the template a bit, but I think it's better now.